Gemfile

```diff
@@ -10,15 +10,17 @@ def location_for(place, fake_version = nil)
   end
 end
 
-beaker_version = ENV['BEAKER_VERSION'] || '>=1.19.0'
 group :development, :test do
   gem 'rake',                    :require => false
   gem 'mocha', '~>0.10.5',       :require => false
   gem 'puppetlabs_spec_helper',  :require => false
-  gem 'serverspec',              :require => false
   gem 'puppet-lint',             :require => false
   gem 'simplecov',               :require => false
   gem 'rspec', '~>2.14.0',       :require => false
+end
+
+beaker_version = ENV['BEAKER_VERSION'] || '~> 2.2'
+group :acceptance do
```

Comment:
Please use :system_tests as this is a pattern that the modules team has already adopted. https://github.com/puppetlabs/puppetlabs-concat/blob/master/Gemfile#L25